api/logic/environment_resource_getter.go

```diff
@@ -151,17 +151,12 @@ func (c *EnvironmentResourceGetter) getPendingTaskResourcesInJobs(environmentID
 				}
 
 				if req.EnvironmentID == environmentID {
-					// note that this isn't exact if the job has started some, but not all of the tasks
-					deployIDCopies := map[string]int{
-						req.DeployID: int(req.Copies),
-					}
-
 					// resource consumer ids are just used for debugging purposes
-					generateID := func(deployID, containerName string, copy int) string {
-						return fmt.Sprintf("Task: %s, Deploy: %s, Container: %s, Copy: %d", req.TaskName, deployID, containerName, copy)
+					generateID := func(deployID, containerName string) string {
```

Comment:
not sure why any of this is changing? copy is fine even if it's always 1. I feel like this is introducing new bugs, would rather we not touch this unless we have to.